draft/XXXX-missing-values-in-typed-cols/README.md

```diff
@@ -0,0 +1,44 @@
+---
+RFC: XXXX
+Author: Jack Wilburn
+Created: 2020-10-26
+Last-Modified: 2020-10-26
+---
+
+# RFC XXXX: Missing Values In Typed Columns
+
+As we move to having typed data in Multinet, it is important to handle missing data elegantly, even with typing. Currently, if a column has any amount of missing data, we assume that the data is a "label" type. Since the "label" type is essentially a catch-all for data that we can't provide a meaningful type for, it's important that if there is a semantically meaningful type, we provide it to visualization applications. This must, also, be true in the case where we have mostly complete data with just a few missing rows. Without this information, the client applications cannot adequately choose a visualization for these columns.
+
+There will likely be some client- and server-side considerations for handling missing data. This will include correctly parsing data and when reading it into the ArangoDB types (or we may choose to leave them as strings in the db). Types that we define, will be referenced as semantic types, and native ArangoDB types will be referenced as Arango types
```

Comment:
```suggestion
There will likely be some client- and server-side considerations for handling missing data. This will include correctly parsing data and when reading it into the ArangoDB types (or we may choose to leave them as strings in the db). Types that we define will be referenced as *semantic types*, and native ArangoDB types will be referenced as *Arango types*.
```

Reply:
Eventually, when an RFC for the column types API appears (attn: @AlmightyYakob), we'll want to link to that RFC from here.